### PR TITLE
test(ui-library): add coverage for UNotifications

### PR DESCRIPTION
**Pull request by drepram**
## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🧪 Tests
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/ui_library/test/widgets/notifications/u_notifications_test.dart [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:ui_library/ui_library_export.dart';
+// import 'package:ui_library/widgets/global/unread_messages_indicator.dart';
+import 'package:ui_library/widgets/notifications/u_notification.dart';
+import 'package:ui_library/widgets/notifications/u_notification_card.dart';
+// part 'package:ui_library/widgets/notifications/models/u_sent_link_notification.dart';
+// part 'package:ui_library/widgets/notifications/models/u_friend_request_notification.dart';
+// part 'package:ui_library/widgets/notifications/models/u_sent_link_notification.dart';
```
Comment (stavares843) [inline packages/ui_library/test/widgets/notifications/u_notifications_test.dart]: do we need this commented out? 🔨

File: packages/ui_library/test/widgets/notifications/u_notifications_test.dart [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            ),
+          ),
+        ),
+      );
+
+      final Size _baseSize = tester.getSize(find.byType(UNotification));
+      expect(_baseSize.width, equals(800.0));
+      expect(_baseSize.height, equals(600.0));
+
+      // final _UFriendRequestNotification _uNotificationCard =
```
Comment (stavares843) [inline packages/ui_library/test/widgets/notifications/u_notifications_test.dart]: do we need this commented out? 🔨

**Review by stavares843 — CHANGES_REQUESTED** [review]

File: packages/ui_library/test/widgets/notifications/u_notifications_test.dart (hunk 2 above)
Comment (drepram) [reply packages/ui_library/test/widgets/notifications/u_notifications_test.dart]: It's a work-in-progress 😄 Lucas has some ideas on how to improve this PR and I'm adding them today before this PR gets merged 😁

File: packages/ui_library/test/widgets/notifications/u_notifications_test.dart (hunk 1 above)
Comment (drepram) [reply packages/ui_library/test/widgets/notifications/u_notifications_test.dart]: Some of them are necessary for an upcoming fix (see other comment 😄 )

Comment (stavares843) [reply packages/ui_library/test/widgets/notifications/u_notifications_test.dart]: thanks 😄

File: packages/ui_library/test/widgets/notifications/u_notifications_test.dart (hunk 2 above)
Comment (stavares843) [reply packages/ui_library/test/widgets/notifications/u_notifications_test.dart]: thanks 😄

**Review by stavares843 — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

File: pubspec.lock [hunk 3]
```diff
@@ -7,7 +7,7 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "31.0.0"
+    version: "43.0.0"
```
Comment (stavares843) [inline pubspec.lock:10]: @lgmarchi are you ok with these updates?

Comment (drepram) [reply pubspec.lock:10]: For me, these updates were necessary because without it... my Flutter was not working :(

**Review by lgmarchi — APPROVED** [review]
The tests are working pretty well, and now it is possible to see some tests in a basic flow. 
image

https://user-images.githubusercontent.com/63157656/183159189-6d276738-80ea-42a3-946f-c7b92e2db034.mov

**Pull request merged**